Stratis.Bitcoin.Features.Consensus/StakeValidator.cs

```diff
@@ -641,19 +261,10 @@ public void CheckKernel(ContextInformation context, ChainedBlock pindexPrev, uin
             }
 
             UnspentOutputs prevUtxo = coins.UnspentOutputs[0];
-            if (IsProtocolV3((int)nTime))
-            {
-                if (this.IsConfirmedInNPrevBlocks(prevUtxo, pindexPrev, this.consensusOptions.StakeMinConfirmations - 1))
-                {
-                    this.logger.LogTrace("(-)[LOW_COIN_AGE]");
-                    ConsensusErrors.InvalidStakeDepth.Throw();
-                }
-            }
-            else
+            if (this.IsConfirmedInNPrevBlocks(prevUtxo, pindexPrev, this.consensusOptions.StakeMinConfirmations - 1))
             {
-                uint nTimeBlockFrom = prevBlock.Header.Time;
-                if (nTimeBlockFrom + this.consensusOptions.StakeMinAge > nTime)
-                    ConsensusErrors.MinAgeViolation.Throw();
```

Comment:
This rule can be deleted? `MinAgeViolation`

Reply:
Ah yes, you mean the error itself, yes, this can be removed and I'll do that as part of PR 549.

Reply:
fixed there